src/UI/Factory.php

```diff
@@ -717,4 +717,25 @@ public function legacy($content);
 	 */
 	public function table();
 
+	/**
+	 * ---
+	 * description:
+	 *   purpose: >
+	 *     Alerts inform the user about the state of the system. Such as the successful completion, the need for further
+	 *     input  of an actual error or stopping users in their tracks in high-risk tasks.
+	 *   composition: >
+	 *     Alerts are embedded or popup messages comprised of text and optionally buttons.
+	 *   effect: >
+	 *     Alerts convey information and optionally provide interaction by using Buttons.
```

Comment:
If you would move that down to the alerts that actually may contain buttons, we would spare rules that again disallow that interactions. Same goes for the interface.

Reply:
I think we need a "may" rule for all types except confirmation where it would be a "must" rule.

Reply:
Fine for me.

Reply:
Done on the top definition level now.